src/core/common/errors/__snapshots__/with-toolkit-error.test.js.snap

```diff
@@ -1,3 +1,25 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`logToolkitError should log a detailed toolkit error the the console 1`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [
+      "Toolkit Error:
+  - Feature: mock feature
+  - Feature Setting: false
+  - Function: observe
+  - Message: mock error",
+      "Error: mock error
+    at Object.<anonymous> (/Users/joshmade/dev/github.com/toolkit-for-ynab/toolkit-for-ynab/src/core/common/errors/with-toolkit-error.test.js:55:21)
+    at Object.asyncFn (/Users/joshmade/dev/github.com/toolkit-for-ynab/toolkit-for-ynab/node_modules/jest-jasmine2/build/jasmine_async.js:124:345)
+    at resolve (/Users/joshmade/dev/github.com/toolkit-for-ynab/toolkit-for-ynab/node_modules/jest-jasmine2/build/queue_runner.js:51:12)
```

Comment:
This snapshot will fail whenever paths change. We should probably exclude this test from snapshots, yeah?

Reply:
Ah yah. This snapshot is deprecated just need to remove it.